applications/GeoMechanicsApplication/tests/cpp_tests/test_GeoMechanicsNewtonRaphsonErosionProcessStrategy.cpp

```diff
@@ -34,9 +34,14 @@ KRATOS_TEST_CASE_IN_SUITE(ErosionProcessStrategy, KratosGeoMechanicsIntegrationS
     auto projectFile = "ProjectParameters.json";
 
     auto execute = KratosExecute();
-    int status = execute.ExecuteFlowAnalysis(workingDirectory, projectFile, 3, 4, 0.1, "PorousDomain.Left_head",
-                                             &flow_stubs::emptyLog, &flow_stubs::emptyProgress,
-                                             &flow_stubs::emptyLog, &flow_stubs::emptyCancel);
+    Kratos::KratosExecute::CriticalHeadInfo critical_head_info(3, 4, 0.1);
+    Kratos::KratosExecute::CallBackFunctions call_back_functions(&flow_stubs::emptyLog,
+                                                                 &flow_stubs::emptyProgress,
+                                                                 &flow_stubs::emptyLog,
+                                                                 &flow_stubs::emptyCancel);
+
+    int status = execute.ExecuteFlowAnalysis(workingDirectory, projectFile, critical_head_info, "PorousDomain.Left_head", call_back_functions);
```

Comment:
Also this int can be const (same holds for the other tests)
```suggestion
    const int status = execute.ExecuteFlowAnalysis(workingDirectory, projectFile, critical_head_info, "PorousDomain.Left_head", call_back_functions);
```